### PR TITLE
Suggested updates

### PR DESCRIPTION
**Pull request by mdarby**
As per @everett-toews

### CODE REVIEW

File: _tutorials/2015-10-26-drupal-and-swarm.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This tutorial provides steps for running a Drupal instance and a MySQL database
 
     `source docker.env`
 
-1. Create an [overlay network](https://docs.docker.com/engine/userguide/networking/get-started-overlay/#step-3-create-the-overlay-network). This virtual network will privately encapsulate your containers for security.
+1. Create an overlay network. This virtual network will privately encapsulate your containers for security.
 
     `docker network create --driver overlay --subnet=10.0.9.0/24 mynetwork`
```
Comment (everett-toews) [inline _tutorials/2015-10-26-drupal-and-swarm.md]: Investigated how overlay networking interacts with ServiceNet a bit today. Needs a bit more investigation tomorrow on what we should recommend people do when they create networks. `--driver overlay` is unnecessary as overlay what is created by default. Need to confirm with the team what (if anything) we should use for `--subnet`.

**Comment by everett-toews** [comment]
@mdarby Can you please update `docker-versions` to 1.10.1? Then this is good to go.

**Comment by everett-toews** [comment]
@mdarby Can you please update `docker-versions` to 1.10.2?

Then this is good to go

**Comment by mdarby** [comment]
Updated :)

**Pull request merged**